### PR TITLE
Use Python 3.7 installed with conda

### PR DESCRIPTION
**Pull request by edwardjkim**
*Issue #, if available:*

*Description of changes:*

**R2**

Removed unnecessary pip download and installation. pip is available in Conda.

*Tests*
```
pytest test/integration
```
Internal container tests and integration tests all passed (except HPO).

**R1**

Since we now use conda to install ml-io, this PR proposes moving to Python 3.7. Previously `apt-get install python3` installed ubuntu 16's default python version 3.5. By removing `apt-get install python3`, the conda-installed Python is used.

*Tests:*
```
pytest test/integration
```
Internal functional tests and integration tests all passed (except HPO).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-patlin** [comment]
LGTM. Are there any concerns with raising the python version?

File: docker/0.90-2/base/Dockerfile.cpu [hunk 1]
```diff
@@ -2,29 +2,38 @@ FROM ubuntu:16.04
...
-RUN conda update -y conda
-RUN conda install -c conda-forge pyarrow=0.14.1
-RUN conda install -c mlio -c conda-forge mlio-py=0.1
+
+RUN conda update -y conda && \
+    conda install -c conda-forge pyarrow=0.14.1 && \
+    conda install -c mlio -c conda-forge mlio-py=0.1
+
+# Install pip
+RUN cd /tmp && \
```
Comment (cbalioglu) [inline docker/0.90-2/base/Dockerfile.cpu]: pip should be available in the default Conda environment. Is there a specific reason why we are explicitly installing pip?

Comment (edwardjkim) [reply docker/0.90-2/base/Dockerfile.cpu]: Right, this was a left over from the previous version. Let me remove it.

**Comment by edwardjkim** [comment]
Closing. Will raise another one.

**Pull request closed without merging**